client/src/test/kotlin/io/spine/examples/pingh/client/e2e/PersonalInteractionTest.kt

```diff
@@ -26,44 +26,115 @@
 
 package io.spine.examples.pingh.client.e2e
 
+import com.google.protobuf.Duration
+import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.matchers.shouldBe
 import io.spine.examples.pingh.client.e2e.given.expectedMentionsList
+import io.spine.examples.pingh.client.e2e.given.randomUnread
 import io.spine.examples.pingh.client.e2e.given.updateStatusById
 import io.spine.examples.pingh.github.Username
 import io.spine.examples.pingh.github.buildBy
+import io.spine.examples.pingh.mentions.MentionId
 import io.spine.examples.pingh.mentions.MentionStatus
+import io.spine.examples.pingh.mentions.MentionView
+import io.spine.protobuf.Durations2.hours
+import io.spine.protobuf.Durations2.milliseconds
+import java.lang.Thread.sleep
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 
 /**
- * End-to-end test that describes such a scenario:
- *
- * 1. The user logs in to the Pingh app.
- * 2. The user updates their mentions from GitHub.
- * 3. The user changes the status of a single mention to snoozing.
- * 4. The user reads one mention.
+ * End-to-end test to checks client-server interaction.
  */
 public class PersonalInteractionTest : IntegrationTest() {
 
-    @Test
-    public fun `the user should log in, update mentions and change their statuses`() {
-        val username = Username::class.buildBy("MykytaPimonovTD")
-        client().logIn(username)
+    private val username = Username::class.buildBy("MykytaPimonovTD")
+    private lateinit var actual: List<MentionView>
+    private lateinit var expected: List<MentionView>
 
+    @BeforeEach
+    public fun logInAndUpdateMentions() {
+        client().logIn(username)
         client().updateMentions()
-        var actual = client().findUserMentions()
-        var expected = expectedMentionsList(username)
+        actual = client().findUserMentions()
+        expected = expectedMentionsList(username)
+        actual shouldBe expected
+    }
+
+    /**
+     * End-to-end test that describes such a scenario:
+     *
+     * 1. The user logs in to the Pingh app.
+     * 2. The user updates their mentions from GitHub.
+     * 3. The user changes the status of a single mention to snoozing.
+     * 4. The user reads one mention.
```

Comment:
How is this point related to pp. 2 and 3? I would understand if we speak of the same mention (first, snoozed, then read). But with this approach, I don't understand the meaning of this scenario.